apps/test/unit/templates/manageStudents/MoveStudentsTest.js

```diff
@@ -51,6 +59,7 @@ describe('MoveStudents', () => {
         {...DEFAULT_PROPS}
         updateStudentTransfer={updateStudentTransfer}
         transferStudents={transferStudents}
+        cancelStudentTransfer={cancelStudentTransfer}
```

Comment:
You might want to compose `DEFAULT_PROPS` and the spy callbacks in your `beforeEach` function, since you pass them in every single test.